CONTRIBUTING.md

```diff
@@ -262,6 +262,15 @@ GUI elements. Keep it simple. Remember, the goal is to show off language feature
 Also, I ask that you don't use external libraries. I like for these files to
 be as easy as possible to test, so limiting dependencies is helpful.
 
+### File Input/Output
```

Comment:
This explains what file IO is, but it doesn't outline what exactly the program should accomplish. In other words, is there a specific thing you'd like the program to accomplish?

Based on your example, I assume we write some arbitrary text to a file and read it back.